templates/sensor_init.tpl

```diff
@@ -0,0 +1,30 @@
+#cloud-config
```

Comment:
We can come back and revisit this with https://github.com/corelight/terraform-config-sensor/pull/1